pandas/tests/indexing/test_datetime.py

```diff
@@ -231,3 +231,21 @@ def test_loc_setitem_with_existing_dst(self):
             dtype=object,
         )
         tm.assert_frame_equal(result, expected)
+
+    @pytest.mark.parametrize(
+        "indexer", [["2001-01", "2001-01-30"], ["2001-01", Timestamp("2001-01-30")]]
+    )
+    def test_loc_getitem_partial_strings_in_list(self, indexer):
+        # GH#27180
+        ser = Series(1, index=date_range("2001-01-29", periods=60))
+        result = ser.loc[indexer]
+        expected = Series(
+            1,
+            index=[
+                Timestamp("2001-01-29"),
+                Timestamp("2001-01-30"),
+                Timestamp("2001-01-31"),
+                Timestamp("2001-01-30"),
+            ],
+        )
```

Comment:
nitpick, but this might look a bit better if you define `index = ` on its own line and then here have `expected = Series(1, index=index)`

Reply:
+1